python-avocado.spec

```diff
@@ -238,6 +238,7 @@ AVOCADO_CHECK_LEVEL=0 selftests/run
 %config(noreplace)%{_sysconfdir}/avocado/avocado.conf
 %config(noreplace)%{_sysconfdir}/avocado/conf.d/README
 %config(noreplace)%{_sysconfdir}/avocado/conf.d/gdb.conf
+%config(noreplace)%{_sysconfdir}/avocado/conf.d/jobscripts.conf
```

Comment:
Can't this be done for all `etc` content?

Reply:
Not sure... but this also helps us to catch errors if we ever copy content to that dir that shouldn't be in there.  We'd get a "file in buildroot but not packaged" kind of error.

Reply:
But you'll get the standard behavior with `rpmnew`, which is imo more important.